imod/tests/test_mf6/test_mf6_wel.py

```diff
@@ -137,35 +141,141 @@ def test_to_mf6_pkg__high_lvl_transient(basic_dis, well_high_lvl_test_data_trans
     np.testing.assert_equal(mf6_ds["rate"].values, rate_expected)
 
 
-def test_clip_box__high_lvl_stationary(well_high_lvl_test_data_stationary):
+class ClipBoxCases:
```

Comment:
Looks better! I'm still missing a test with an unstructured grid, you can easily convert a structured grid to an unstructured grid with: 
``idomain_ugrid = xu.UgridDataArray.from_structured(idomain)``